### PR TITLE
Elasticache, add parameter group name

### PR DESCRIPTION
**Pull request by rahulmehrotra1**
Added a new parameter parameter_group_name to specify a non-default parameter_group_name to be used

### CODE REVIEW

**Comment by rahulmehrotra1** [comment]
Fixes #436

File: cloud/amazon/elasticache.py [hunk 1]
```diff
@@ -69,6 +69,16 @@
       - A list of cache security group names to associate with this cache cluster
     required: false
     default: ['default']
+  parameter_group_name:
+    description: 
+      - Specify non-default parameter group names to be associated with cache cluster
+    requiredL false
```
Comment (denmat) [inline cloud/amazon/elasticache.py:75]: That looks like a typo.

**Comment by bennojoy** [comment]
@rhimhrtr could you please resolve conflicts and squash to a single commit.

**Comment by gregdek** [comment]
@rhlmhrtr A friendly reminder: this pull request has been marked as needing your action, and it hasn't been updated in the last month. 

If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we'll keep it open pending your changes.

If we receive no indication from you within a week, we will close this pull request.

**Comment by gregdek** [comment]
Closed due to inactivity.

**Pull request closed without merging**